plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -479,6 +481,12 @@ public boolean collectionExists(MongoDatabase db, String collectionName)
         return false;
     }
 
+    private boolean indexExists(MongoCollection<Document> schemaCollection) {
```

Comment:
`{` should be the next line. 
https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#code-style